app/services/question-service.server.ts

```diff
@@ -0,0 +1,26 @@
+import type { QuestionstatsResponse } from "~/types/generated/QuestionStateController";
+import type BountyQuestionService from "./contracts/bounty-question-service.server";
+import type QuestionStateControllerService from "./contracts/question-state-controller-service.server";
+
+export default class QuestionService {
+  constructor(
+    private bountyQuestionService: BountyQuestionService,
+    private questionStateControllerService: QuestionStateControllerService
+  ) {}
+
+  //TODO: use this
+  async getQuestions() {
+    const currentQuestion = await this.bountyQuestionService.getCurrentQuestion();
+    if (!currentQuestion) {
+      return [];
```

Comment:
HMMM I'm think we should fail or give an error message if we can't get current question on the all questions page without that we really can't get all questions